tests/test_handlers.py

```diff
@@ -91,18 +90,26 @@ def setUp(self):
         super(TestLogin, self).setUp()
         self.factory.org.set_setting('auth_password_login_enabled', True)
 
-    @classmethod
```

Comment:
I *think* those methods are leftovers from a previous iteration of the multi org feature.